hl7/Module.md

```diff
@@ -78,47 +78,18 @@ public function main() returns error? {
 }
 ```
 
-## Classes
-
-### HL7Client
-
-Represents an isolated client that can be used to send HL7 messages to a remote server.
+#### Configuring Secure Connection
+To configure TLS for the HL7 client, you can pass a `secureSocket` configuration in the `tcp:ClientConfiguration` when initializing the client. This allows secure communication with the HL7 server.
 
 ```ballerina
-public isolated client class HL7Client {
-    // Client implementation
-}
-```
-
-#### Methods
-
-##### `init`
-```ballerina
-public isolated function init(string remoteHost, int remotePort) returns hl7v2:HL7Error?
-```
-
-Initializes the HL7 client with the given remote host and port.
-
-###### Parameters
-- `remoteHost`: Remote host to connect to.
-- `remotePort`: Remote port to connect to.
-
-###### Return Value
-- `hl7v2:HL7Error?`: An error if initialization fails.
-
-##### `sendMessage`
```

Comment:
Are these removals intentional?

Reply:
These are generated in the Bal doc generation, no need to replicate IMO

Reply:
+1